snippets/quantity-input.liquid

```diff
@@ -11,7 +11,7 @@
 {% endcomment %}
 
 <quantity-input class="quantity cart-quantity">
-  <button class="quantity__button" name="minus" type="button">
+  <button class="quantity__button" name="minus" type="button" data-target="decrement-{{ variant.id }}">
```

Comment:
`data-target` enables us to refocus inputs after content is updated so keyboard order doesn't get goofed up